UPGRADING.md

```diff
@@ -6,6 +6,13 @@ While we specify a version of Lemmy, pict-rs, postgres, etc. at the point in tim
 
 While you are not forced into running the specific versions, we do not go through thorough testing on all version compatibility matrices, so please make your best judgement and always backup before performing updates.
 
+### Upgrading to 1.5.6 (Lemmy 0.19.10)
+
+#### Steps
+
+- `git pull && git checkout 1.5.6`
+- Run your regular deployment. Example: `ansible-playbook -i inventory/hosts lemmy.yml --become`
+
```

Comment:
I wouldnt bother copy-pasting the same upgrade instructions for every version. Its not helpful and makes it harder to find the versions with manual steps required.

Reply:
It's necessary because of the release tag.

Reply:
You mean the `git checkout *version*`? It would be enough to put that at the top as a general instruction, without repeating it every time.

Reply:
I spose we could, but many tagged versions will have to have their own specific upgrade instructions anyway.